### PR TITLE
fix: [OBI-157] added video call list view and add overlay

### PR DESCRIPTION
**Pull request by web-mi**
Part of #423 
Fixes #OBI-157

## Proposed Changes

  - Implemented first version of waiting room
  - Implemented videocall appointments Listing with CRUD functions
  - Implemented video call integration routes and videocall

### CODE REVIEW

File: src/resources/scripts/i18n/de/videoConferenceInformal.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const videoCoferenceInformal = {
+	'waitingroom.subline':
+		'Der Video-Call hat noch nicht begonnen. Du wirst weitergeleitet sobald dein/e Berater/in den Video-Call startet.'
```
Comment (koepferd) [inline src/resources/scripts/i18n/de/videoConferenceInformal.ts]: For the open source core please always use the gender gap: _

File: src/resources/scripts/i18n/de/videoConference.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		'Bitte bestätigen Sie unsere Datenschutzbestimmungen.',
+	'waitingroom.dataProtection.description':
+		'Danach dürfen unsere Berater_innen einen Chat mit Ihnen starten.',
+	'waitingroom.dataProtection.label':
+		'Ich habe die %legal_links% zur Kenntnis genommen. Für Authentifizierung und Navigation verwendet diese Webseite Cookies. Damit erkläre ich mich einverstanden.',
+	'waitingroom.dataProtection.label.and': 'und',
+	'waitingroom.dataProtection.button': 'Bestätigen',
+	'waitingroom.headline': 'Bitte haben Sie etwas Geduld',
+	'waitingroom.subline':
+		'Der Video-Call hat noch nicht begonnen. Sie werden weitergeleitet sobald ihr/e Berater/in den Video-Call startet.',
```
Comment (koepferd) [inline src/resources/scripts/i18n/de/videoConference.ts]: For the open source core please always use the gender gap: _

File: src/components/appointment/Appointment.tsx [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+export const Appointment = ({
+	appointment,
+	editClick,
+	deleteClick
+}: AppointmentProps) => {
+	const [overlayItem, setOverlayItem] = useState(null);
+	const [showMore, setShowMore] = useState(false);
+
+	const startVideoCallButton: ButtonItem = {
+		label: 'Videocall starten',
```
Comment (koepferd) [inline src/components/appointment/Appointment.tsx]: Please use a translation key here.

File: src/components/appointment/Appointments.tsx [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+		icon: <CameraPlusIcon />
+	};
+
+	const editAppointment = useCallback((appointment) => {
+		setOverlayItem(onlineMeetingFormOverlay(setOnlineMeeting, appointment));
+	}, []);
+
+	const deleteAppointment = useCallback(
+		(appointment) => {
+			// ToDo: Implement API call
```
Comment (koepferd) [inline src/components/appointment/Appointments.tsx]: Remove comment please, as it is implemented

**Review by koepferd — DISMISSED** [review]

**Review by alxbenz — APPROVED** [review]

**Pull request merged**